accord-core/src/main/java/accord/impl/SimpleProgressLog.java

```diff
@@ -822,7 +825,7 @@ void ensureScheduled()
                 return;
 
             isScheduled = true;
-            node.scheduler().once(() -> commandStore.execute(PreLoadContext.empty(), ignore -> run()).begin(commandStore.agent()), 200L, TimeUnit.MILLISECONDS);
+            node.scheduler().once(() -> commandStore.execute(PreLoadContext.empty(), ignore -> run()).begin(commandStore.agent()), config.progress_log_scheduler_delay_in_ms, TimeUnit.MILLISECONDS);
```

Comment:
we are trying to move away from `_in_ms` postfix... this may call out that we should likely move some of the config classes to accord?